tools/ctl/firestore/firestore.go

```diff
@@ -277,8 +277,10 @@ func (f *Client) FetchRebuilds(ctx context.Context, req *FetchRebuildRequest) (r
 		rebuilds[r.ID()] = r
 	}
 	if err := <-cerr; err != nil {
-		log.Fatal("query error", err.Error())
+		err = errors.Wrap(err, "query error")
+		return nil, err
 	}
+	log.Printf("Fetched %d rebuilds", len(rebuilds))
```

Comment:
this log line might go better in the caller function. not necessary but i'd generally lean towards moving log messages as far up as possible.

Reply:
Moved.